plugins/services/src/js/structs/Service.js

```diff
@@ -96,6 +96,12 @@ module.exports = class Service extends Item {
     };
   }
 
+  isDeleting() {
```

Comment:
Same as above.

Reply:
Why aren’t we adding the to the state? Feels strange to add a dedicated getter for this one.